### PR TITLE
Correct error strings in glusterfs

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@k8s-bot ok to test

File: pkg/volume/glusterfs/glusterfs.go [hunk 1]
```diff
@@ -309,7 +309,7 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 
 	p := path.Join(b.glusterfs.plugin.host.GetPluginDir(glusterfsPluginName), b.glusterfs.volName)
 	if err := os.MkdirAll(p, 0750); err != nil {
-		return fmt.Errorf("glusterfs: mkdir failed: %v", err)
```
Comment (jsafrane) [inline pkg/volume/glusterfs/glusterfs.go]: Why is the directory in square brackets?

`glusterfs: mkdir [/var/lib/kubelet/plugins/kubernetes.io/glusterfs/xyz] failed: Permission denied`.

What about `Error creating directory /var/lib/kubelet/plugins/kubernetes.io/glusterfs/xyz: Permission denied`?

I don't like `glusterfs` prefix everywhere too, but that's probably for another PR.

Comment (humblec) [inline pkg/volume/glusterfs/glusterfs.go]: @jsafrane Sure.. I think `Error creating..." is better, I will change to that format.

**Comment by humblec** [comment]
@jsafrane Addressed the comment, Thanks!

**Pull request merged**